PriceQuoteIos/Utils/APIHandler.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHandler.swift
+//  PriceQuoteIos
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class APIHandler {
+    public static func handleAPIResponse<T: BaseResponse> (data: DataResponse<T>,
+                                                           completion: @escaping (BaseCallback<T>) -> Void) {
+        switch(data.result) {
+        case .success(let response):
+            switch response.status {
+            case 201:
```

Comment:
Actually If I'm not wrong, Alamofire API provides a method called `validate` which you can do previous validation based on the response, for things like status code and even content type (if it a JSON response or not, things of that nature)

Reply:
Removed it.